src/System.Management.Automation/engine/remoting/commands/CustomShellCommands.cs

```diff
@@ -4808,6 +4808,101 @@ public sealed class EnablePSRemotingCommand : PSCmdlet
     Add-Content -Path $pluginConfigFile -Value ""CORECLRDIR=$PSHOME""
 }}
 
+function Copy-PluginToEndpoint
+{{
+param(
+    [Parameter()] [string] $endpointDir
+)
+    $resolvedPluginInstallPath = """"
+    $pluginInstallPath = Join-Path ""$env:WINDIR\System32\PowerShell"" $endpointDir
+    if (!(Test-Path $pluginInstallPath))
+    {{
+        $resolvedPluginInstallPath = New-Item -Type Directory -Path $pluginInstallPath
+    }}
+    else
+    {{
+        $resolvedPluginInstallPath = Resolve-Path $pluginInstallPath
+    }}
+    if (!(Test-Path $resolvedPluginInstallPath\{5}))
+    {{
+        Copy-Item $PSHOME\{5} $resolvedPluginInstallPath -Force
+        if (!(Test-Path $resolvedPluginInstallPath\{5}))
+        {{
+            Write-Error ($errorMsgUnableToInstallPlugin -f ""{5}"", $resolvedPluginInstallPath)
+            return $null
+        }}
+    }}
+    return $resolvedPluginInstallPath
+}}
+
+function Register-Endpoint
+{{
+param(
+    [Parameter()] [string] $configurationName
+)
+    #
+    # Section 1:
+    # Move pwrshplugin.dll from $PSHOME to the endpoint directory
+    #
+    $pwshVersion = $configurationName.Replace(""PowerShell."", """")
```

Comment:
It is not clear to me why "PowerShell." is removed from the configuration name.  Can you add examples in the comment section that illustrates the expected behavior, i.e., before string then after string?

Reply:
I wanted to stick to the original version of the code where the plugin path was: '$env:WINDIR\System32\PowerShell\' + powershell_version (for the tied to the version endpoint it would be $PSVersionTable.PSVersion). To be consistent for our new endpoint the plugin path is: '$env:WINDIR\System32\PowerShell\6'.
I added the comment in code.